modules/ossm-rn-deprecated-features.adoc

```diff
@@ -15,6 +15,28 @@ Deprecated functionality is still included in {product-title} and continues to b
 
 Removed functionality no longer exists in the product.
 
+== Deprecated and removed features {SMProductName} 2.3.1
+
+The `concurrency` field of the `spec.proxy` API is deprecated as of release 2.3.1, and will be removed in a future release. While the field is still supported in release 2.3.1, customers should start using the `concurrency` field of the `ProxyConfig` API.
+
+The `concurrency` field specifies the number of worker threads to run. If the field is set to 0, then the number of worker threads available is equal to the number of CPU cores. If the field is not set, then the number of worker threads available defaults to 2.
+
+In the following example, the `concurrency` field is set to 0.
+
+[source,yaml]
+----
+apiVersion: networking.istio.io/v1beta1
+kind: ProxyConfig
+metadata:
+  name: per-workload-proxyconfig
+  namespace: example
+spec:
+  selector:
```

Comment:
let's remove the whole `selector` block from here for now. The deprecated setting was global, so the example should also apply globally - we could maybe mention that users can use the selectors to restrict it to a specific set of workloads, but it's not necessarily required

Reply:
done